packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -77,6 +88,33 @@ export abstract class Metric<T> implements types.Metric<T> {
     return;
   }
 
+  /**
+   * Provides a ReadableMetric with one or more TimeSeries.
+   * @returns The ReadableMetric, or null if TimeSeries is not present in
+   *     Metric.
+   */
+  get(): ReadableMetric | null {
+    if (this._handles.size === 0) return null;
+
+    const timestamp = hrTime();
+    return {
+      descriptor: this._metricDescriptor,
+      timeseries: Array.from(this._handles, ([_, handle]) =>
+        ((handle as unknown) as BaseHandle).getTimeSeries(timestamp)
```

Comment:
Why is there a need to cast here? Should we have the template parameter above be `T extends BaseHandle` since that seems implied? Could that remove the need to cast?

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-js/pull/437/commits/9bef4bf40944b1743185afc4466b77bdd2de13a2, PTAL.